client/src/routes/index.js

```diff
@@ -1,5 +1,5 @@
 export const routes = {
-    Home: '/',
-    Login: '/login',
-    SignUp: '/signup'
+    Login: '/',
```

Comment:
@Ireti203 Why is the landing page the login page? I don't think it should be that way.